content/chapter/03.tex

```diff
@@ -1,47 +1,201 @@
 
-Erste Erwähnung eines Akronyms wird als Fußnote angezeigt. Jede weitere wird
-nur verlinkt: \acf{AGPL}. \cite{fsf:2007}
+Das größte Problem stellt die Kombination eines \ac{9-DOF} Sensors und \ac{IR}s an einem einzigen Arduino dar.
+\begin{figure}[H]
+	\centering
+	\includegraphics[width=0.9\textwidth]{images/chapter/03/hardware-layout-with-one-arduino.png}
+	\caption{Altes Hardwaremodell mit einem Arduino}
+\end{figure}
+Bei gleichzeitigem Messen beider Sensoren ist eine zuverlässige Detektion des Rotorblattes nicht möglich.
+Abbildung \ref{fig:interrupts_are_shit} stellt die Messergebnisse des \ac{IR}s graphisch dar.
+\begin{figure}[H]
+	\centering
+	\includegraphics[width=0.9\textwidth]{images/chapter/03/interrupts_are_shit.png}
+	\caption{Messwerte des \ac{IR}s bei gleichzeitiger Messung eines \ac{IR}s und eines \ac{9-DOF}s}
+	\label{fig:interrupts_are_shit}
+\end{figure}
+Wie in Abbildung \ref{fig:interrupts_are_shit} zu erkennen ist, wird das Propellerblatt nur unregelmäßig erkannt.
+Des Weiteren werden teilweise 3 Interrupts durch die \ac{ISR} erkannt.
+
+Aus diesem Grund erfolgt eine Aufteilung der Sensoren auf die zwei Arduinos \ac{A1} und \ac{A2} wie in Abbildung \todo{ref auf hardwaremodell bild setzen} dargestellt. 
+Weiterhin erfolgt eine Aufgabenteilung, wie in Abbildung \ref{fig:aufgaben-arduinos} dargestellt.
+\begin{figure}[H]
+	\centering
+	\includegraphics[width=0.9\textwidth]{images/chapter/03/aufgaben-arduinos.png}
+	\caption{Aufgaben der beiden Arduinos \ac{A1} und \ac{A2}}
+	\label{fig:aufgaben-arduinos}
+\end{figure}
+Durch die Trennung beider Sensoren und die Aufgabenteilung in Propellerblattdetektion und Unwuchtmessung ist \ac{A1} nun in der Lage das Propellerblatt konstant (im Rahmen einer gewissen Fehlertoleranz) zu detektieren.
+Für ein besseres Verständnis, kann der Programmablauf von \ac{A1} dem Anhang \ref{lst:arduino_selfmade_interrupts} entnommen werden.
+\begin{figure}[H]
+	\centering
+	\includegraphics[width=0.9\textwidth]{images/chapter/03/self_made_interrupt_a1.png}
+	\caption{Messerte des \ac{A1}}
+	\label{fig:self_made_interrupt_a1}
+\end{figure}
+
+\subsection{Detektion der Unwucht}
+\label{subsec:detektion_der_unwucht}
+
+\subsubsection*{Theoretische Vorgehensweise}
+Besitzt ein Propellerblatt eine Unwucht, erzeugt dieser deutlich spührbare Vibrationen am Motorblock.
+Diese Vibrationen sollen mithilfe eines 3-Achsen Sensors gemessen werden.
+Je nach Auslenkung des Motorblockes kann hierdurch eine Unwucht detektiert werden.
+Desweiteren kann unter Berücksichtigung der Propellerdetektion, eine Unwuchtlokalisierung erfolgen. 
+
+Um solch eine Unwucht zu erzeugen, werden wie in Abbildung \ref{fig:propeller-mit-unwucht} gezeigt, mehrere Propeller mit unterschiedlichen Gewichten, bzw. identischen Gewichten jedoch an unterschiedlichen Stellen präpariert.
```

Comment:
UM eine UNwucht ... vielleicht besser: Zur Erzeugung einer Unwucht werden, wie in Abb (..) gezeigt, mehrere ....
und wieder das jedoch...